src/goto-programs/read_goto_binary.h

```diff
@@ -19,25 +19,33 @@ class goto_modelt;
 class message_handlert;
 class symbol_tablet;
 
+// Read given goto binary
+// Does not update config
```

Comment:
Could the documentation please be placed with the definition and use Doxygen style?